test/e2e/storage/drivers/csi.go

```diff
@@ -893,6 +893,12 @@ func (g *gcePDCSIDriver) PrepareTest(ctx context.Context, f *framework.Framework
 		return cfg
 	}
 
+	// Check if the cluster is already running gce-pd CSI Driver
+	deploy, err := f.ClientSet.AppsV1().Deployments("gce-pd-csi-driver").Get(ctx, "csi-gce-pd-controller", metav1.GetOptions{})
```

Comment:
The driver may not always be installed via Deployment. What do you think of an env variable to control this?

Reply:
The manifests over here https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/deploy/kubernetes/README.md install the controller as a Deployment. 

I don't think we need that. Our test suite should check if you deployed the components using the "official" manifests.